On unit testing the input configurations of Model_Net, 5 of 8 cases failed with OOM error.
Run the command: `pytest tests/ttnn/unit_tests/operations/test_conv_transpose2d.py::test_model_net_conv_t2d`
